addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -522,7 +522,7 @@ def grouping_key_generator(base_line, tax_values):
                 'id': invoice.name,
                 'issue_date': invoice.invoice_date,
                 'due_date': invoice.invoice_date_due,
-                'note_vals': [{'note': html2plaintext(invoice.narration)}] if invoice.narration else [],
+                'note_vals': [{'note': self._format_note(invoice.narration)}] if invoice.narration else [],
```

Comment:
Why do we need to call it via an extra function? It seems to be exactly the same 🤔